apps/admin-x-activitypub/src/hooks/use-activity-pub-queries.ts

```diff
@@ -79,39 +79,11 @@ const QUERY_KEYS = {
         return ['thread', id];
     },
     feed: ['feed'],
-    inbox: ['inbox']
+    inbox: ['inbox'],
+    postsByAccount: ['posts_by_account'],
+    postsLikedByAccount: ['posts_liked_by_account']
```

Comment:
How about just `account_posts` and `account_likes`? (similar to `account_follows` above)